src/docfx/build/page/PageBuilder.cs

```diff
@@ -286,15 +289,38 @@ private JObject LoadSchemaDocument(ErrorBuilder errors, FilePath file)
                 JsonUtility.Merge(pageModel, new JObject { ["metadata"] = userMetadata.RawJObject });
             }
 
+            var mime = _documentProvider.GetMime(file);
+
             // Validate and transform model using JSON schema
-            var content = _jsonSchemaTransformer.TransformContent(errors, file);
+            var (content, token) = _jsonSchemaTransformer.TransformContent(errors, file);
             if (!(content is JObject transformedContent))
             {
                 throw Errors.JsonSchema.UnexpectedType(new SourceInfo(file, 1, 1), JTokenType.Object, content.Type).ToException();
             }
+            else if (token is JObject fileContent)
+            {
+                switch (mime.Value?.ToLowerInvariant())
+                {
+                    case Constants.LearningPath:
```

Comment:
We don't use `Constants` in docfx, if this string needs sharing, it probably indicates some kind of code smell.
```suggestion
                    case "learningpath":
```

Reply:
OK